src/main/java/Yukee.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+
+public class Yukee {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
```

Comment:
Try to minimize the use of magic numbers. Instead, you could do the following:
```suggestion
        private static final int MAX_LENGTH = 100;
        Task[] tasks = new Task[MAX_LENGTH];
```